libraries/c_sdk/standard/mqtt/test/system/iot_tests_mqtt_system.c

```diff
@@ -211,6 +211,40 @@ static bool _disconnectSerializerOverride = false;  /**< @brief Tracks if #_disc
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Establish an MQTT connection. Retry if retries are enabled.
+ */
+static IotMqttError_t _mqttConnect( const IotMqttNetworkInfo_t * pNetworkInfo,
+                          const IotMqttConnectInfo_t * pConnectInfo,
+                          uint32_t timeoutMs,
+                          IotMqttConnection_t * const pMqttConnection )
+{
+    IotMqttError_t status = IOT_MQTT_STATUS_PENDING;
+#if ( IOT_TEST_MQTT_CONNECT_RETRY_ENABLED == 1)
+    int32_t retried = 0;
+    uint32_t periodMs = 200;
+    int32_t retries = 3;
```

Comment:
Why is 3 the right number of retries?

Reply:
There is no particular reason for this number other than it feeling good